gitlab/base.py

```diff
@@ -32,6 +32,11 @@
 ]
 
 
+_URL_ATTRIBUTE_ERROR = (
+    "https://python-gitlab.readthedocs.io/en/stable/faq.html#attribute-error-list"
```

Comment:
Just thinking, maybe we could use a permalink to make sure the user gets the exact link? Sometimes RTD is a bit odd about updating versions or maybe I missed something. I just checked and had to manually change the branch from master to main as well.

```suggestion
    f"https://python-gitlab.readthedocs.io/en/v{__version__}/faq.html#attribute-error-list"
```
(will need an extra import though)

Reply:
Done